pkg/features/kube_features.go

```diff
@@ -252,6 +258,9 @@ var defaultVersionedFeatureGates = map[featuregate.Feature]featuregate.Versioned
 	TASProfileMixed: {
 		{Version: version.MustParse("0.11"), Default: false, PreRelease: featuregate.Deprecated},
 	},
+	RemoveFinalizersWithStrictPatch: {
+		{Version: version.MustParse("0.11"), Default: false, PreRelease: featuregate.Beta},
```

Comment:
Please also add [here](https://github.com/kubernetes-sigs/kueue/blob/b6793cffac91b081a7728985b255ef847fc53525/site/content/en/docs/installation/_index.md?plain=1#L262) and update KEP.